### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by shariefyoussef**
I created a basic README file based on GYBITG website.

### CODE REVIEW

**Review by Juanitaag1 — APPROVED** [review]
Looks good. A suggestion would be to add in the first paragraph, the purpose of the app from our project documentation. I think James sent you the link since we created it back before you became a part of our team. Here is what we had for the purpose:
Get Your Butt in the Gym is an app that supports female basketball players in their junior and senior years of high school. It allows the girls to showcase their athletic skills, as well as develop and track essential basketball skills that college coaches are looking for in a new recruit. 
Nice job. Thanks!

**Review by jc-hayes — COMMENTED** [review]
From my understanding, a README file is written more for developers, so I don't think we need the "who we are", "what we do", "how we do it", "about coach bobbie" sections. I think a little more detail about the actual app would be useful, including the features that are done, and then ones that still need to be built out. Here is a link to the updated backlog that includes all that stuff: https://docs.google.com/document/d/1RqTkZqdZkeXhb4zJsLHf_s5y5OY3aBvseSZC4WZssOQ/edit#

**Comment by shariefyoussef** [comment]
I added the purpose Juanita provided and made the changes that James suggested.

**Review by jc-hayes — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# gybitg-ios
+
+GYBITG
+
+Get Your Butt in the Gym is an app that supports female basketball players in their junior and senior years of high school. It allows the girls to showcase their athletic skills, as well as develop and track essential basketball skills that college coaches are looking for in a new recruit.
+
+Some of the current features include recording and save videos, creating an account, and adding ew game stats.
```
Comment (jc-hayes) [inline README.md]: Found a slight typo "ew" -> "new"

Comment (Bansenauer-Cascadia) [reply README.md]: also "recording and saving videos"

**Review by Bansenauer-Cascadia — APPROVED** [review]

**Pull request merged**